collections/Laud_Misc/MS_Laud_Misc_146.xml

```diff
@@ -39,42 +42,158 @@
+                     <bibl>Among 9 MSS. listed by Ottman, op. cit., at p. 386.</bibl>
+                     <textLang mainLang="la">Latin</textLang>
+                  </msItem>
+                  <msItem n="6" xml:id="MS_Laud_Misc_146-item6">
+                     <locus>(fols. 119r–130v) </locus>
+                     <author key="person_4472713">Simon de Cassia</author>
+                     <title key="work_6666">De gestis domini salvatoris</title><bibl> (excerpt: lib. X, cap. 31)</bibl>
+                     <rubric>De commemoratione</rubric>
+                     <incipit>Itaque ista erit aptissima regula ad propositum principale puta ad demonstrandum quod hii inquirentes utilitatem plebis</incipit><explicit>qui dat vitam mundo per Ihesum Christum dominum nostrum Amen</explicit>
+                     <finalRubric>Expliciunt determinaciones doctorum de crebra communione</finalRubric>
+                     <textLang mainLang="la">Latin</textLang>
+                  </msItem>
+                  <msItem n="7" xml:id="MS_Laud_Misc_146-item7">
+                     <locus>(fols. 131r–140v) </locus>
+                     <author key="person_160152177">Konrad von Dryburg</author>
+                     <title key="work_6669">Determinatio de celebratione missae</title>
+                     <rubric>Incipit determinacio .C. presbiter de celebratione missarum</rubric>
+                     <incipit><supplied>D</supplied>eus in adiutorium meum intende Scribitur in Ps lxix <gap reason="editorial"/> Reverendi patres domini et magistri cum non virtute sua fragilitas humana</incipit>
+                     <explicit>ad aliquas horas peculiares vel vigilias mortuorum obligatur</explicit>
+                     <finalRubric>Sic est finis C. presbiter de cele<ex>bratione</ex> missarum summatim repetiti solempniter per Conradum ordinarum <unclear>doc<ex>torem</ex></unclear> Erfordensis. Amen</finalRubric>
+                     <textLang mainLang="la">Latin</textLang>
+                  </msItem>
+                  <msItem n="8" xml:id="MS_Laud_Misc_146-item8">
+                     <locus>(fols. 140v–152r)</locus>
+                     <author key="person_207244871">Henricus de Langenstein</author>
+                     <title key="work_6667">De reductione effectuum spiritualium in virtutes communes</title>
+                     <incipit>Propter Ammirari inceperunt antiquitus homines prophetari</incipit>
+                     <explicit>nos custodiat Marie filius Ihesus Cristus amen</explicit>
+                     <finalRubric>Hec ille - - - Explicit edificium magistri Heinrici de Hassia limpidissimi speculatoris deo laus</finalRubric>
```

Comment:
Is `- - -` an illegible word?

Reply:
I wondered that as well - it represents a sort of squiggle in the manuscript. Other ideas for representing this?

Reply:
If you need a solid bar, there's the three-em dash: ⸻.

Reply:
yes, that would probably be clearer

Reply:
I suppose you could also add a `<sic>` tag to clarify that this is not editorial.